001-cron.php

```diff
@@ -99,5 +123,11 @@ function wpcom_vip_disable_jetpack_sync_on_cron_shutdown( $load_sync ) {
 	 */
 	add_filter( 'jetpack_sync_sender_should_load', 'wpcom_vip_disable_jetpack_sync_on_cron_shutdown' );
 
+	/**
+	 * Log details of events that fail
+	 */
+	add_action( 'a8c_cron_control_event_threw_catchable_error', 'wpcom_vip_log_cron_control_event_for_caught_error', 10, 2 );
+	add_action( 'a8c_cron_control_freeing_event_locks_after_uncaught_error', 'wpcom_vip_log_event_object' );
```

Comment:
Now that I look at these action names again, feels like the tenses are a bit off. Probably would have been better named `a8c_cron_control_event_throw_catchable_error` (`threw` => `throw`) and `a8c_cron_control_free_event_locks_after_uncaught_error` (`freeing` => `free`).